### PR TITLE
[RFR] Introduced a new way to execute and configure error pages

### PR DESCRIPTION
**Pull request by mickaelandrieu**
I moved the exception management to a ``kernel.exception`` listener.

Also, imo we should be able to override the error templates because our clients always want a customised error page, with their own content and dependencies.

This is an actual POC, but I need a review and advices to improve my commits.

Finally, how can I test it ?

### CODE REVIEW

File: Event/Listener/ExceptionListener.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+     * @var BackBee\Renderer
+     */
+    private $renderer;
+
+    /**
+     * @var Symfony\Component\HttpFoundation\Response
+     */
+    private $response;
+
+    public function __construct(BBApplication $bbapp)
```
Comment (fkroockmann) [inline Event/Listener/ExceptionListener.php]: $bbapp -> $application (better :))

File: Event/Listener/ExceptionListener.php [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    {
+        $this->bbapp = $bbapp;
```
Comment (fkroockmann) [inline Event/Listener/ExceptionListener.php]: same here

File: Event/Listener/ExceptionListener.php [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    }
+
+    /**
+     * Returns the path of the template for the selected HTTP Code error status
+     *
+     * @input string $error 404|500
+     *
+     * @return string
+     */
+    private function getErrorTemplate($error)
```
Comment (fkroockmann) [inline Event/Listener/ExceptionListener.php]: $errorName ?

File: Resources/layouts/error/500.phtml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (fkroockmann) [inline Resources/layouts/error/500.phtml]: French words ?

File: Resources/layouts/error/404.phtml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (fkroockmann) [inline Resources/layouts/error/404.phtml]: French words ?

Comment (mickaelandrieu) [inline Resources/layouts/error/404.phtml]: yeah :/ as before. Do we have a translator in PHP side ?

Comment (fkroockmann) [inline Resources/layouts/error/404.phtml]: I Don't know :/

Comment (mickaelandrieu) [inline Resources/layouts/error/404.phtml]: Ok for translations but in another PR as I want to improve the system to allow the choice of format (xliff, yaml, php array, etc).

For now, all templates are in french, so *for consistency* I provide french templates.

**Comment by mickaelandrieu** [comment]
ping @eric-chau. This is now done, translations done will be done in another PR ( when I will make the translator able to use every loader from Symfony translator component).

**Pull request merged**